truss/truss_config.py

```diff
@@ -372,3 +349,52 @@ def clone(self):
     def validate(self):
         for secret_name in self.secrets:
             validate_secret_name(secret_name)
+
+
+dataclass_to_req_keys_map = {
```

Comment:
Should probably make a constant (ALL_CAPS) and move to the top

Reply:
Can't move it to the top since all the classnames are not defined yet.